app/views/results_entities.php

```diff
@@ -26,9 +26,10 @@
     $path_locale2 = VersionControl::getPath($locale, $current_repo, $entity);
 
     // Escape strings for HTML display
```

Comment:
This comment probably need to be updated?

Reply:
Moving